pytorch_lightning/trainer/training_loop.py

```diff
@@ -250,7 +249,7 @@ class TrainerTrainLoopMixin(ABC):
     on_validation_end: Callable
 
     @abstractmethod
-    def get_model(self) -> LightningModule:
```

Comment:
Why'd you remove the type hint?

Reply:
i didn't on purpose... not sure why it's gone. let me fix